department-of-veterans-affairs/vets-website
# VAGOV-3779: add field alert to health care region detail pages

## Description
Allow reusable alerts and featured content at the top of detail pages

## Testing done
Locally with staging data

1. rebuild app with staging data (at this time i have a test page with alerts on it)
2. browse to http://localhost:3001/pittsburgh-health-care/test-detail-page-alerts/
3. check out those alerts

## Screenshots
![localhost_3001_pittsburgh-health-care_test-detail-page-alerts_](https://user-images.githubusercontent.com/19178435/58856307-2a367700-8657-11e9-9fba-6eee0d3eb8be.png)

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-3779

## Definition of done
